README.md

```diff
@@ -95,6 +95,8 @@ func main() {
 }
 ```
 
+For futher examples see the [EXAMPLES.md](./EXAMPLES.md) document.
```

Comment:
Can we add this link as well to the documentation section above?

Reply:
Done!